### PR TITLE
Changelog for 2.4.0

### PR DESCRIPTION
**Pull request by daschuer**
That was quite a but of work. We have now > 900 enties in the 2.4 changelog. 
For a release that will be too long, but for the beta it is IMHO ok. 

Not sure how much we need to meld it down, because we also have the blog post for a condensed version. 

Has one interest to prepare the release version of his change log by  removing the issues and PRs that are not interesting from a user perspective.

### CODE REVIEW

File: res/linux/org.mixxx.Mixxx.metainfo.xml [hunk 1]
```diff
@@ -569,7 +674,30 @@
...
+   #11418
+  </li>
+  <li>
+   Verify the "first sound" of as an analysis sanity check
+   #4773
+  </li>
+  <li>
+   Fix a possible crash when ejecting a track #11334](
+   https://github.com/mixxxdj/mixxx/pull/11334
+   )
```
Comment (ronso0) [inline res/linux/org.mixxx.Mixxx.metainfo.xml:694]:
```suggestion
   Fix a possible crash when ejecting a track
   #11334
```

File: res/linux/org.mixxx.Mixxx.metainfo.xml [hunk 2]
```diff
@@ -1261,6 +1525,17 @@
...
+   #10927
+   #11063
+   #11153
+   #11180
+   #11167
+   #11181
+   #11266
+   #11312
+   [#11389](
+   https://github.com/mixxxdj/mixxx/pull/11389
```
Comment (ronso0) [inline res/linux/org.mixxx.Mixxx.metainfo.xml:1538]:
```suggestion
   #11389
```

**Comment by ronso0** [comment]
> Has one interest to prepare the release version of his change log by removing the issues and PRs that are not interesting from a user perspective.

Sure, can do that. We can group a lot of UX fixes for example, and some minor bugfixes in different categories.

**Review by ronso0 — CHANGES_REQUESTED** [review]
Pre-commit complains about plain-text urls

File: .codespellignore [hunk 3]
```diff
@@ -6,3 +6,4 @@ sord
 doubleClick
 sur
 jus
+euclidian
```
Comment (fwcd) [inline .codespellignore:9]: It's spelled "euclidean", thus probably shouldn't be excluded

Comment (Swiftb0y) [reply .codespellignore:9]: Yep, sorry my mistake. We should correct the title and the changelog entry of the corresponding PR instead.

Comment (Swiftb0y) [reply .codespellignore:9]: Yep, sorry my mistake. We should correct the title and the changelog entry of the corresponding PR instead.

**Comment by daschuer** [comment]
rebase on 2.4 to fix the faulty commits and the failing workflow run.

**Comment by JoergAtGithub** [comment]
We should mention, that the mappings for

- Novation Twitch
- Korg Nanokontrol 2
- MAudio Xponent

were removed in this release (see https://github.com/mixxxdj/mixxx/pull/2682) due to incompatibility of the old code with the new mapping system.

**Comment by ronso0** [comment]
Great you remembered that.
See also #10690

**Comment by daschuer** [comment]
Done

**Comment by ronso0** [comment]
Thank you!

**Pull request merged**